.nvmrc

```
@@ -1 +1 @@
-12
+16
```

It probably should not be included:

```suggestion
12
```